template.yaml

```diff
@@ -154,11 +156,21 @@ Resources:
                 - dynamodb:DeleteItem
               Resource: !GetAtt PromptTable.Arn
       Events:
-        POST:
+        GetAllPrompts:
+          Type: Api
+          Properties:
+            Method: GET
+            Path: /get-all-prompts
```

Comment:
nit: If you wanted to make this API definition more _rest_like, you could try the following structure:
[helpful blog](https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/#h-use-nouns-instead-of-verbs-in-endpoint-paths)
```
GET /prompts -> Returns list of prompts
GET /prompts/{id} -> Return specific prompt
PUT /prompts/{id} -> Update a specific prompt
POST /prompts -> Create a new prompt
```

Not sure where `evaluate-translation` would fit in though. 😅